spec.html

```diff
@@ -3900,6 +3899,11 @@ <h1>Well-Known Intrinsic Objects</h1>
     <h1>ECMAScript Specification Types</h1>
     <p>A specification type corresponds to meta-values that are used within algorithms to describe the semantics of ECMAScript language constructs and ECMAScript language types. The specification types include Reference, List, Completion Record, Property Descriptor, Environment Record, Abstract Closure, and Data Block. Specification type values are specification artefacts that do not necessarily correspond to any specific entity within an ECMAScript implementation. Specification type values may be used to describe intermediate results of ECMAScript expression evaluation but such values cannot be stored as properties of objects or values of ECMAScript language variables.</p>
 
+    <emu-clause id="sec-enum-specification-type">
+      <h1>The Enum Specification Type</h1>
+      <p><dfn variants="enum,enums">Enums</dfn> are values which are internal to the specification and not directly observable from ECMAScript code. Enums are denoted using a ~sans-serif~ typeface. For instance, a Completion Record's [[Type]] field takes on values like ~normal~, ~return~, or ~throw~. The content of an enum serves no purpose other than to distinguish it from other enums, and implies nothing about its usage or meaning in context.</p>
```

Comment:
> Enums are values which are internal to the specification and not directly observable from ECMAScript code.

While this is true, it's not very good as a definition, because it's true of *every* specification type value.

> The content of an enum ...

Enums have content? I'd be inclined to say the *name* of an enum.

Reply:
> While this is true, it's not very good as a definition, because it's true of _every_ specification type value.

I was mostly just taking it from the existing wording. Added "Enums have no characteristics other than their name".

> Enums have content? I'd be inclined to say the _name_ of an enum.

Sure. Done.